components/screens/menu/AboutScreen.tsx

```diff
@@ -25,13 +26,7 @@ export const AboutScreen = (_: NativeStackScreenProps<MenuStackParamList, 'about
   const openUrl = useCallback(({data}: {data: string}) => void WebBrowser.openBrowserAsync(data), []);
   const copyVersionInfoToClipboard = useCallback(() => {
     void (async () => {
-      await Clipboard.setStringAsync(
-        `Avy version ${Application.nativeApplicationVersion || 'n/a'} (${Application.nativeBuildVersion || 'n/a'})
-Git revision ${process.env.EXPO_PUBLIC_GIT_REVISION || 'n/a'}
-Update group ID ${updateGroupId} (channel: ${Updates.channel || 'development'})
-Update ID ${Updates.updateId || 'n/a'} (platform: ${Platform.OS})
-User ID ${mixpanelUserId || 'n/a'}`,
```

Comment:
@yuliadub seems like we could make `getVersionInfoFull()` accept the MixPanel ID as `string | undefined` and inside of the function, do this `mixpanelUserId || 'n/a'` transform to handle the case where it's missing.